src/back/handlers.py

```diff
@@ -0,0 +1,30 @@
+"""Module containing handlers."""
+import dataclasses
+from datetime import datetime
+from typing import Self
+
+from src.back.interfaces.handlers import MessageHandlerInterface
+from src.back.interfaces.io import WriteStreamInterface
+from src.back.interfaces.message import MessageInterface
+from src.back.interfaces.room_registry import RoomRegistryInterface
+from src.back.room import Room
+
+
+@dataclasses.dataclass
+class CreateRoomMessage(MessageInterface):
+    """Create room message."""
+
+    name: str
+    created_at: datetime
+
+
+@dataclasses.dataclass
+class CreateRoomHandler(MessageHandlerInterface[CreateRoomMessage]):
```

Comment:
4) And this is where i declare what type of message I am working with when subclassing from handler interface

Reply:
Yes